presentation_parser.py

```diff
@@ -0,0 +1,69 @@
+from pptx.enum.shapes import MSO_SHAPE_TYPE
+from pptx import Presentation
+
+
+def extract_text_from_shape(shape) -> str:
```

Comment:
Separate code to small functions.  Read more: https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be